golang/cli_test/test_helpers.go

```diff
@@ -0,0 +1,423 @@
+package clitest
+
+import(
+    "testing"
+	"encoding/json"
+    "os"
+    "fmt"
+    "io/ioutil"
+    "path/filepath"
+    "strings"
+    "github.com/stretchr/testify/require"
+    app "github.com/summa-tx/relays/golang"
+    rtypes "github.com/summa-tx/relays/golang/x/relay/types"
+
+	"github.com/cosmos/cosmos-sdk/tests"
+    clientkeys "github.com/cosmos/cosmos-sdk/client/keys"
+    "github.com/cosmos/cosmos-sdk/crypto/keys"
+    "github.com/cosmos/cosmos-sdk/server"
+    sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const (
+	keyFoo       = "foo"
+	keyBar       = "bar"
+    stakeDenom   = "stake"
+    fooDenom     = "footoken"
+)
+
+var startCoins = sdk.NewCoins(
+    sdk.NewCoin(stakeDenom, sdk.TokensFromConsensusPower(2000000)),
+    sdk.NewCoin(fooDenom, sdk.TokensFromConsensusPower(2000000)),
+)
+
+// Fixtures is used to setup the testing environment
+type Fixtures struct {
```

Comment:
this is good, but look up testify suites:

https://godoc.org/github.com/stretchr/testify/suite

And how we've used them here:
https://github.com/summa-tx/bitcoin-spv/blob/master/golang/btcspv/bitcoin_spv_test.go#L88-L116

Reply:
See my last commit `use suite module` to see how I incorporated `testify/suite` I used it to DRY up json_data. Each test still needs it's own fixture though if they're to run in parallel.